docs/using/api.md

```diff
@@ -17,5 +33,228 @@ interface for exploring the API is available at:
+```json
+{
+  "jobs": [
+    {
+      "job_parameters": {
+        "granules": [
+          "S1B_IW_GRDH_1SDV_20210210T153157_20210210T153222_025546_030B48_2901"
+        ],
+        {
+          "radiometry": "gamma0",
+          "scale": "power",
+          "dem_matching": false,
+          "include_dem": true,
+          "include_inc_map": true,
+          "include_scattering_area": false,
+          "speckle_filter": false
+        }
+      },
+      "job_type": "RTC_GAMMA",
+      "name": "custom-rtc-example"
+    },
+    {
+      "job_parameters": {
+        "granules": [
+          "S1B_IW_GRDH_1SDV_20210210T153132_20210210T153157_025546_030B48_4E31"
+        ],
+        {
+          "radiometry": "sigma0",
+          "scale": "amplitude",
+          "dem_matching": false,
+          "include_dem": false,
+          "include_inc_map": false,
+          "include_scattering_area": true,
+          "speckle_filter": true
+        }
+      },
+      "job_type": "RTC_GAMMA",
+      "name": "custom-rtc-example"
+    }
+  ]
+}
+```
+
+### Submitting Sentinel-1 InSAR jobs
+
+You can also submit InSAR jobs for scene pairs using [ESA granule IDs](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/naming-conventions).
+```json
+{
+  "jobs": [
+    {
+      "job_parameters": {
+        "granules": [
+          "S1A_IW_SLC__1SDV_20200203T172103_20200203T172122_031091_03929B_3048",
+          "S1A_IW_SLC__1SDV_20200110T172104_20200110T172123_030741_03864E_A996"
+         ]
+      },
+      "job_type": "RTC_GAMMA",
+      "name": "minimal-insar-example"
+    },
+    {
+      "job_parameters": {
+        "granules": [
+          "S1A_IW_SLC__1SDV_20200527T195012_20200527T195028_032755_03CB56_3D96",
+          "S1A_IW_SLC__1SDV_20200515T195012_20200515T195027_032580_03C609_4EBA"
+         ],
+        "looks": "10x2",
+        "include_look_vectors": true,
+        "include_los_displacement": true
+      },
+      "job_type": "RTC_GAMMA",
+      "name": "custom-insar-example"
+    }
+  ]
+}
+```
+
+### Submitting autoRIFT jobs
+
+AutoRIFT support processing Sentinel-1, Sentinel-2, or Landsat-8 Collection 2 pairs.
+
+* Sentinel-1 jobs are submitted using [ESA granule IDs](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/naming-conventions)
+* Sentinel-2 jobs can be submitted using [ESA granule IDs](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-2-msi/naming-convention)
+  or [Element 84 Earth Search IDs](https://registry.opendata.aws/sentinel-2/)
+* Landsat-8 Collection 2 jobs are submitted using [USGS scene IDs](https://www.usgs.gov/faqs/what-naming-convention-landsat-collection-2-level-1-and-level-2-scenes?qt-news_science_products=0#qt-news_science_products)
+
+To submit an example set of jobs including all supported missions, you could write a job list like:
+
+```json
+{
+  "jobs": [
+    {
+      "name": "autorift-example",
```

Comment:
I initially looked for `name` at the bottom of the job based on the previous examples, I'd consider using the same order across all examples, maybe with `name` and `job_type` preceding `job_parameters`

Reply:
good call